### DISCUSSION TITLE
How do I set which database the Search function searches?

### DISCUSSION THREAD

**Discussion by piotrek-k** [dbeaver/dbeaver#23148] [Q&A]
This thing confuses me a lot and I can't figure out how to do it properly.

There is a nice feature called "Search" which appears when you press Ctrl+H. It allows me to search through table names. It worked well until I connected to SQL server that hosts multiple databases. When I press Ctrl+H, "Search" window opens but there is no way to choose which database I want to search on. Then I press the "Search" button and it searches one of my databases, but I'm not sure how it determines which one.

For example, I've got 3 databases - X, Y and Z - and each of them has a table called "Accounts". When I search for "Accounts", I can only see results from the X database.

**Comment by LonwoLonwo** [comment]
Hello @piotrek-k 
The Search dialog looks like this usually:

![2024-03-16 20_46_13-Search](https://github.com/dbeaver/dbeaver/assets/45152336/bbedd59e-18b6-422c-8850-54cb9cf301da)

You can choose the database there. Like in my case, when I select the codeCompileTest element, that means that the search will be in this database. 
You can check the query in the [Query Manager](https://github.com/dbeaver/dbeaver/wiki/Query-Manager) (with enabled filters)
